experimental/logging/log_listener.go

```diff
@@ -47,7 +47,18 @@ func (f *loggingListenerFactory) NewListener(fnd api.FunctionDefinition) experim
 		!exported { // not callable by the host
 		return nil
 	}
-	return &loggingListener{writer: f.writer, fnd: fnd, isWasi: fnd.ModuleName() == "wasi_snapshot_preview1"}
+
+	// special-case formatting of WASI error number until there's a generic way
```

Comment:
there's one wasi function which has no result, and future change will allow us to see multiple results. so I made the errno field a position instead of a boolean.